env.common.yml

```diff
@@ -9,19 +9,19 @@ dependencies:
   - pip
   - pre-commit=2.10.*
   - pymatgen=2020.12.31
-  - python=3.8.*
-  - pytorch=1.9.0
+  - python=3.9.*
+  - pytorch=1.10.1
   - pyyaml=5.4.*
   - tensorboard=2.5.*
   - tqdm=4.58.*
   - sphinx
   - nbsphinx
   - pandoc
-  - black
+  - black==22.3.0
   - numba
   - pip:
     - Pillow
-    - torch-geometric==1.7.2
+    - git+https://github.com/pyg-team/pytorch_geometric.git@a7e6be4
```

Comment:
What's the reason to point to this specific commit as opposed to a more stable release (e.g. 2.0.4)?

Reply:
discussed offline - 2.0.4 has this bug for DimeNet(++) https://github.com/pyg-team/pytorch_geometric/pull/4424